sdk/attestation/azure-security-attestation/README.md

```diff
@@ -257,63 +254,51 @@ This example shows one common pattern of calling into the attestation service to
-    # When the 'quote' (attestation evidence) is created specify the SHA256 hash
-    # of the runtime data when creating the evidence.
-    #
-    # When the generated evidence is created, the hash of the runtime data is
-    #  included in the secured portion of the evidence.
-    #
-    # The Attestation service will validate that the Evidence is valid and that
-    # the SHA256 of the RuntimeData parameter is included in the evidence.
-    response = attest_client.attest_sgx_enclave(
-            quote,
-            runtime_data=AttestationData(runtime_data, is_json=False))
-    assert response.value.enclave_held_data == runtime_data
-    assert response.value.sgx_collateral is not None
-
-    # At this point, the EnclaveHeldData field in the attestationResult.Value
-    # property will hold the input binaryRuntimeData.
-
-    # The token is now passed to the "relying party". The relying party will
-    # validate that the token was issued by the Attestation Service. It then
-    # extracts the asymmetric key from the EnclaveHeldData field. The relying
-    #  party will then Encrypt it's "key" data using the asymmetric key and
-    # transmits it back to the enclave.
-    encryptedData = send_token_to_relying_party(attestationResult.Token)
-
-    # Now the encrypted data can be passed into the enclave which can decrypt that data.
+response, token = attest_client.attest_sgx_enclave(quote, runtime_data=runtime_data)
 ```
 
+At this point, the enclave_held_data attribute in the attestationResult
+will hold the input binary runtime_data.
+
+The token is now passed to the "relying party". The relying party will
+validate that the token was issued by the Attestation Service. It then
+extracts the asymmetric key from the EnclaveHeldData field. The relying
+party will then Encrypt its "key" data using the asymmetric key and
+transmits it back to the enclave.
+
+```python
+encrypted_data = send_token_to_relying_party(attestationResult.Token)
+```
+
+Now the encrypted data can be passed into the enclave which can decrypt that data.
+
 Additional information on how to perform attestation token validation can be found in the [MAA Service Attestation Sample](https://github.com/Azure-Samples/microsoft-azure-attestation).
 
 ### Retrieve Token Certificates
 
 Use `get_signing_certificates` to retrieve the certificates which can be used to validate the token returned from the attestation service.
 
 ```python
-    signers = attest_client.get_signing_certificates()
-    for signer in signers:
-        cert = cryptography.x509.load_der_x509_certificate(signer.certificates[0], backend=default_backend())
-        print('Cert  iss:', cert.issuer, '; subject:', cert.subject)
+signers = attest_client.get_signing_certificates()
+for signer in signers:
+    cert = cryptography.x509.load_pem_x509_certificate(signer.certificates[0].encode('ascii'), backend=default_backend())
+    print('Cert  iss:', cert.issuer, '; subject:', cert.subject)
```

Comment:
What is `default_backend()`? Can you add the import for where that is found?

Reply:
default_backend() is an annoying python legacy artifact that is required for older versions of cryptography. Newer versions default the backend parameter to default_backend so customers don't have to import it.